src/AzureAuth/DoubleExtensions.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.AzureAuth
+{
+    using System;
+
+    /// <summary>
+    /// Extensions to <see cref="double"/>.
+    /// </summary>
+    internal static class DoubleExtensions
+    {
+        /// <summary>
+        /// Create a <see cref="TimeSpan"/> of n minutes.
+        /// </summary>
+        /// <param name="n">Number of minutes.</param>
+        /// <returns><see cref="TimeSpan"/> with n minutes.</returns>
+        public static TimeSpan Minutes(this double n)
+        {
+            return TimeSpan.FromMinutes(n);

```

Comment:
Is it worth it to have this extension? This seems like a very Rubyish thing to do here (though I suppose we've taken inspiration from other languages already). Personally, I'm not convinced we need this, but I could go either way if other feel like this is worthwhile.